kubernetes-elasticsearch-cluster/deploy-es.sh

```diff
@@ -12,22 +12,17 @@ then
 fi
 
 dataset=$1
-project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
-gcloud config set project ${project_id}
 
-# Need to get cluster name by sorting the list of clusters, and choosing to
-# use the one with the greatest timestamp (most recent)
-cluster_line=$(gcloud container clusters list | grep elasticsearch-cluster- | sort -rn -k1 | head -n1)
-cluster_name=$(echo $cluster_line | awk '{print $1}')
-zone=$(echo $cluster_line | awk '{print $2}')
+util/setup-gcloud.sh ${dataset}
+project_id=$(kubectl config current-context | cut -d "_" -f 2)
+zone=$(kubectl config current-context | cut -d "_" -f 3)
 
 bold=$(tput bold)
 normal=$(tput sgr0)
-echo "Deploying Elasticsearch in cluster ${bold}$cluster_name${normal} for" \
-  "${bold}dataset" "$dataset${normal} in ${bold}project $project_id${normal}"
 echo
-
-gcloud container clusters get-credentials ${cluster_name} --zone ${zone}
+echo "Deploying Elasticsearch for ${bold}dataset $dataset${normal} in" \
```

Comment:
I think having the cluster_name printed is useful for debugging. Would it be ok to put the cluster_name back in here (and in deploy-indexer.sh)? You can do `cluster_name=$(kubectl config current-context | cut -d "_" -f 4)` to get it and be consistent with how you got the other variables.

Reply:
![1](https://user-images.githubusercontent.com/10929390/55433170-a3c3c300-5549-11e9-9959-d377c7ddfcda.png)

Cluster name is still there, it's just not bold.

Cluster name is only useful if I have the GKE page open that lists all the clusters. So I thought it didn't need to be bold...

Reply:
Sounds good, thanks